### DISCUSSION TITLE
Scaling NABox or NE Harvest 2.0

### DISCUSSION THREAD

**Discussion by coreymr** [NetApp/harvest#760] [Q&A]
Hi, in parallel to my other inquiry I was wondering if you had a reference as to how well either NABox or Harvest 2.0 scales?  We are a geo-dispersed org with 130+ filers in multiple regions/countries with varying degrees of connectivity.  

Our OCUMs are setup with regional instances.  Should we think about setting up regional NAbox as well to limit "overloading" the Harvest/Grafana functions?  Or should be okay using one "pane of glass" for our environment?

**Comment by rahulguptajss** [comment]
1. one data point for Harvest 2.0 - there are customers monitoring 300 nodes on a single instance of Harvest running on a machine with 24 GB of memory. You can try scaling it further and see.
2. @ybizeul can provide more information on NABox
